.eslintrc

```diff
@@ -1,12 +1,24 @@
 {
-  "extends": "airbnb-base/legacy",
```

Comment:
I actually removed this because it only covered ES5 standards. I'd like to replace it with the full base library (which covers ES6+) if you are fine with that.